regression/test.pl

```diff
@@ -71,7 +71,9 @@ ($$$$$)
 
   $options =~ s/$ign//g if(defined($ign));
 
-  my $output = $input;
+  my $descriptor = $test;
+  $descriptor =~ s/^.*\///;
```

Comment:
@LAJW not everyone reads Perl, and I can't really Google that string of punctuation, can I? I've an inkling that it's probably related to `sed`'s `s///` operator, but why make me slog through that process for a quarter hour when you could tell me what you're doing in 10 seconds?